src/site/_includes/components/Img.js

```diff
@@ -93,6 +92,21 @@ const Img = function (args) {
     imgTag = html`<a href="${fullSrc}">${imgTag}</a>`;
   }
 
+  if (caption) {
+    imgTag = html`${imgTag}
+      <figcaption class="w-figcaption">
```

Comment:
I think this is a bit of an issue. We don't use the `w-figcaption` class on d.c.c. (I think we just style figcaption directly).

I think we could do it and it wouldn't hurt anything but I'd like to clean it up eventually. It seems like web.dev should just style `figcaption`.